### PR TITLE
Newfaq

### PR DESCRIPTION
**Pull request by a3nm**
Hi all,

I went over the FAQ to implement what was suggested at the general assembly (encourage users to report errors to our data sources rather than to us, when appropriate). I also fixed a bunch of other things. Could you have a look, and merge if you agree?

### CODE REVIEW

File: templates/dissemin/faq.html [hunk 1]
```diff
@@ -326,23 +330,61 @@ <h2 id="statistics">{% trans "Should I trust your statistics?" %}</h2>
...
+            of your co-authors asked us to delete it? Or it may be a bug. In
+            this case, please file an <a
+               href="https://github.com/dissemin/dissemin/issues">issue</a>
+            describing the problem, or send a message to <a
+              href="mailto:bugs@dissem.in">bugs@dissem.in</a>.
             {% endblocktrans %}
+            </p>
+            <p>
+            {% blocktrans trimmed %}
+            Note that, it the longer term, we hope to design a way for our users
```
Comment (Lysxia) [inline templates/dissemin/faq.html]: **in** the longer term?

Comment (wetneb) [reply templates/dissemin/faq.html]: indeed, fixing

Comment (a3nm) [reply templates/dissemin/faq.html]: thanks!

**Comment by a3nm** [comment]
Thanks!

-- 
Antoine Amarilli

**Pull request merged**